yi-frontend-vty/src/Yi/Frontend/Vty.hs

```diff
@@ -103,23 +126,36 @@ data FrontendState = FrontendState
 -- If this is set to its default (None) it will be replaced by the default
 -- vty configuration from standardIOConfig. However, standardIOConfig
 -- runs in the IO monad so we cannot set the real default here.
```

Comment:
Since `defaultConfig` isn't in `IO`, it might be worth reworking this, but probably not while we still support vty 5.